README.md

```diff
@@ -57,13 +57,17 @@ Commands:
 You can use caching by including this module in your ActiveRecord models,
 
     class OfferEvent < ActiveRecord::Base
-      include Gris::CacheKey
+      include Gris::Caching
     end
 
 and then cache inside GET requests like this
 
     offer_event =  OfferEvent.cached_find(id)
 
+To expire the cache for an object, call
+
+    OfferEvent.expire_cache(id)
```

Comment:
We could provide both, but requiring a load of an object in order to expire it may not be needed sometimes, such as expiring `has_many` objects.